pkgs/top-level/all-packages.nix

```diff
@@ -7529,7 +7529,9 @@ let
 
   simgear = callPackage ../development/libraries/simgear { };
 
-  sfml_git = callPackage ../development/libraries/sfml { };
+  sfml = sfml22;
```

Comment:
IMO, it's ok to leave just `sfml` and `sfml_git`. No need to specify `22` additionally.